TeslaSUI/TeslaSUI/Views/ClimateView.swift

```diff
@@ -0,0 +1,393 @@
+                .font(.system(size: 17, weight: .semibold))
+                .frame(width: 50)
+            Button {
+                
+            } label: {
+                Image(Constants.Images.autoIconImageName)
+                    .resizable()
+                    .modifier(ButtonImageModifire())
+            }
+            .neumorphismUnSelectedCircleStyle()
+            Slider(value: $climateViewModel.autoParameters,
+                   in: Constants.Integer.sliderRange,
+                   step: 1)
+            .frame(width: 220)
+        }
+    }
+    
+    private var heatClimateSettingView: some View {
+        HStack {
+            Text(Constants.Text.heat)
+                .font(.system(size: 17, weight: .semibold))
+                .frame(width: 50)
+            Button {
+                
+            } label: {
+                Image(Constants.Images.heatIconImageName)
+                    .resizable()
+                    .modifier(ButtonImageModifire())
+            }
+            .neumorphismUnSelectedCircleStyle()
+            
+            Slider(value: $climateViewModel.heatParameters,
+                   in: Constants.Integer.sliderRange,
+                   step: 1)
+            .frame(width: 220)
+        }
+    }
+    
+    private var trimCirlceProgress: some View {
+        Circle()
+            .trim(from: 0, to: climateViewModel.circleProgress / 15)
+            .stroke(selectColorToGradient(), style: StrokeStyle(lineWidth: 25,
+                                                             lineCap: CGLineCap.round))
+            .frame(height: 180)
+            .rotationEffect(.degrees(-90))
+            .animation(.linear, value: climateViewModel.circleProgress)
+            .padding(5)
+            .neumorphismUnSelectedStyle()
+    }
+    
+    private var screenNameTextView: some View {
+        Text(Constants.Text.climateScreenName)
+            .font(.system(size: 20, weight: .semibold))
+    }
+    
+    private var backButtonView: some View {
+        Button {} label: {
+            Image(Constants.Images.backChevronImageName)
+                .resizable()
+                .scaledToFit()
+                .foregroundColor(.gray)
+                .frame(width: 20, height: 20)
+        }
+        .neumorhismNavigationCircleButtonUnselected()
+    }
+    
+    private var settingsButtonView: some View {
+        Button {
+            climateViewModel.isAlertShown.toggle()
+        } label: {
+            Image(Constants.Images.gearshapeImageName)
+                .resizable()
+                .scaledToFit()
+                .frame(width: 20, height: 20)
+            
+        }
+        .neumorhismNavigationCircleButtonUnselected()
+        .overlay(
+        Circle()
+            .stroke(gradient, lineWidth: 2)
+            .opacity(climateViewModel.isAlertShown ? 1 : 0)
+        )
+    }
+    
+    private var actualTemperature: Int { Int(15.0 + climateViewModel.circleProgress) }
```

Comment:
```suggestion
    private var actualTemperature: Int { Int(15.0 + climateViewModel.circleProgress) }
```
кажется это должно быть во вьюмодели